### PR TITLE
list jobs with default prefix *

### PR DESCRIPTION
**Pull request by tiantn**
Signed-off-by: tian na <tiantn@cn.ibm.com>

### CODE REVIEW

File: __tests__/__system__/cli/list/job/__scripts__/command/command_list_job_noprefix.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+bright zos-ftp list jobs
```
Comment (dkelosky) [inline __tests__/__system__/cli/list/job/__scripts__/command/command_list_job_noprefix.sh:4]: this probably should be `zowe zos-ftp list jobs` - the `bright` high level command name still works, but it generally not used any longer.

**Comment by dkelosky** [comment]
Thanks @tiantn - this works for me and addresses what I was after with #70

**Review by dkelosky — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+- Support list jobs without default prefix *.
```
Comment (std4lqi) [inline CHANGELOG.md:6]: `Support listing jobs ... ` might be better.

**Review by std4lqi — APPROVED** [review]

**Comment by std4lqi** [comment]
One minor comments about wording. Thanks, @tiantn !

File: CHANGELOG.md (hunk 2 above)
Comment (tiantn) [reply CHANGELOG.md:6]: > Thanks @tiantn - this works for me and addresses what I was after with #70

Thanks @dkelosky and @std4lqi!

**Review by zFernand0 — APPROVED** [review]

**Review by lauren-li — CHANGES_REQUESTED** [review]
Overall, this PR looks good to me. Manual testing worked well. All unit and most system tests are passing. Once the snapshot below is updated, I will be happy to approve:

Screen Shot 2021-07-06 at 10 23 47 AM

Thanks @tiantn for this addition!

**Comment by tiantn** [comment]
Thanks @lauren-li , I updated the snapshot but the check failed again. Hi @zFernand0, could you help to solve it? Thank you very much!

**Comment by zFernand0** [comment]
> Thanks @lauren-li , I updated the snapshot but the check failed again. Hi @zFernand0, could you help to solve it? Thank you very much!

Seems like it was the same thing as before. Hopefully once this PR is merged, subsequent branches should adopt the Jenkisnfile node change.

**Comment by tiantn** [comment]
> > Thanks @lauren-li , I updated the snapshot but the check failed again. Hi @zFernand0, could you help to solve it? Thank you very much!
> 
> Seems like it was the same thing as before. Hopefully once this PR is merged, subsequent branches should adopt the Jenkisnfile node change.

Thank you @zFernand0 ! 😊

**Review by lauren-li — APPROVED** [review]
This PR looks good to me now. All unit and system tests are now passing. Thanks @tiantn for addressing my comments, and thanks @zFernand0 for helping out with the checks!

**Comment by lauren-li** [comment]
@std4lqi This looks ready to merge and release whenever you are ready. Thank you!

**Comment by std4lqi** [comment]
Hi @lauren-li , Sorry, I missed it. I don't have authority to merge PR. Usually, Fernando takes care of the code merge and release. 

Hi @zFernand0, could you help merge the code? Thanks in advance!

**Comment by tiantn** [comment]
Hi @zFernand0 , could you help to merge the code but don't release it? Because I will submit another PR and I think it can release when the new PR also merged. Thank you!

**Comment by zFernand0** [comment]
hey @std4lqi  and @tiantn, you should both have access to merge PRs to this repository as of this moment.
I forgot that after the Roles within Zowe were rolled out, some repositories needed specific people to have access.
As for publishing a new version, it seems that `liangqi` is a pipeline admin. (which I believe it was you @std4lqi)
https://github.com/zowe/zowe-cli-ftp-plugin/blob/3f6bb83c5d73f271d33beaa13a8ea0209afbccec/Jenkinsfile#L28

This mean that you should be getting an email with a URL to approve a version number or abort the release in this case.
I'll suggest that @tiantn is also added to the list of pipeline admins for the zFTP pipeline
https://wash.zowe.org:8443/view/CLI/job/zowe-cli-ftp-plugin/

Since there seems to be no rush to merge this PR, why not add yourself to the pipeline admins now and abort the release once the PR is merged by you or Qi 😋 

Please let us know if you are still having difficulties with permissions.

**Comment by tiantn** [comment]
Hi @zFernand0, thank you very much! I accepted your innvitation for collaborating the repository. Thank you! 😊

**Pull request merged**